openupgrade_scripts/scripts/account/14.0.1.1/noupdate_changes.xml

```diff
@@ -1,115 +1,18 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
-  <record id="account_analytic_line_rule_billing_user" model="ir.rule">
-    <field name="perm_create"/>
-    <field name="perm_read"/>
-    <field name="perm_unlink"/>
-    <field name="perm_write"/>
+  <record id="base.default_user" model="res.users">
+    <field name="groups_id" eval="[(4,ref('account.group_account_manager'))]"/>
   </record>
-  <record id="account_bank_statement_comp_rule" model="ir.rule">
-    <field name="global"/>
+  <record id="base.group_portal" model="res.groups">
+    <field name="implied_ids" eval="[(4, ref('account.group_show_line_subtotals_tax_excluded'))]"/>
   </record>
-  <record id="account_bank_statement_line_comp_rule" model="ir.rule">
-    <field name="global"/>
+  <record id="base.group_public" model="res.groups">
+    <field name="implied_ids" eval="[(4, ref('account.group_show_line_subtotals_tax_excluded'))]"/>
   </record>
-  <record id="account_comp_rule" model="ir.rule">
-    <field name="global"/>
+  <record id="base.group_user" model="res.groups">
+    <field name="implied_ids" eval="[(4, ref('account.group_show_line_subtotals_tax_excluded'))]"/>
   </record>
-  <record id="account_fiscal_position_comp_rule" model="ir.rule">
-    <field name="global"/>
-  </record>
-  <record id="account_invoice_line_rule_portal" model="ir.rule">
-    <field name="domain_force">[('move_id.move_type', 'in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund')), ('move_id.message_partner_ids','child_of',[user.commercial_partner_id.id])]</field>
```

Comment:
For sure it is a mistake that sections like these are removed. This rule is updated with the renaming of field `type` to `move_type`. Compare 13 https://github.com/odoo/odoo/blob/13.0/addons/account/security/account_security.xml#L209 with 14 https://github.com/odoo/odoo/blob/14.0/addons/account/security/account_security.xml#L214

Reply:
Oh, then you did it manually, right?

Reply:
No, I did not edit the files after generating them last time.

Reply:
It' very strange :S

Reply:
Sure this means there is a glitch somewhere.

Reply:
Possibly related: https://github.com/OCA/OpenUpgrade/pull/2481#discussion_r616535367

Reply:
Fuck, that's not the problem! haha I did a new run with files instead of paths (adding the UnicodeDecodeError exception) and I still get this wrong noupdate changes. I will soon do a more detailed debug and check the root of the bug.

Reply:
Fascinating! Looking forward for this mystery to unravel.

Reply:
Solved the mystery!! :dancers: